.github/workflows/actions.yml

```diff
@@ -1,55 +1,58 @@
+name: Automated Workflow for Glacier dataset
```

Comment:
@gradedSystem pls change name according to the dataset name

Reply:
fixed